### PR TITLE
Add background color for tabs

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Screenshot 2020-06-07 at 13 51 41
Screenshot 2020-06-07 at 13 51 52

This may be obsolete when we merge https://github.com/spotify/backstage/pull/1076 and start using it instead.

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by benjdlambert** [comment]
@stefanalund typescript woes.

**Comment by stefanalund** [comment]
@benjdlambert should be fixed now.

File: packages/core/src/layout/HeaderTabs/index.tsx [hunk 1]
```diff
@@ -18,12 +18,13 @@
...
 import React from 'react';
-
 import { makeStyles, Tabs, Tab } from '@material-ui/core';
+import { BackstageTheme } from '@backstage/theme';
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles<BackstageTheme>(theme => ({
   tabsWrapper: {
     gridArea: 'pageSubheader',
+    backgroundColor: theme.palette.tabsBackground,
```
Comment (Rugvip) [inline packages/core/src/layout/HeaderTabs/index.tsx:27]: use `theme.palette.background.paper` instead

**Review by Rugvip — CHANGES_REQUESTED** [review]

**Comment by stefanalund** [comment]
@Rugvip good suggestions! Fixed.

**Review by Rugvip — APPROVED** [review]

**Pull request merged**